common/ActionManager.js

```diff
@@ -25,12 +47,11 @@ class ActionManager {
   defineActions(definitions) {
     Object.keys(definitions).forEach(name => {
       const definition = definitions[name];
-      const composed = function() {
-        return compose([
+      const composed = (...args) =>
```

Comment:
Did you mean to refer to this line? I don't see a destructured assignment used here (e.g., `let [a, b] = [1, 2];`)